app/helpers/selections_helper.rb

```diff
@@ -1,2 +1,50 @@
+require 'json'
+require 'ostruct'
+
 module SelectionsHelper
+
+  def get_forecast(stations)
```

Comment:
This `stations` variable is not used but the method call in the controller is doing the right thing by passing in `@selections`. Change this variable name to `selections` and use it on line 14.